task1/Calculator.ts

```
@@ -0,0 +1,38 @@
+export class Calculator {
+
+    addition(x, y): number {
+
+        if (typeof x !== 'number' || typeof y !== 'number') {
+            return NaN;
```

In case of invalid type, it's better to throw an error.